message_chunker.py

```diff
@@ -0,0 +1,33 @@
+import nltk
+
+def message_chunk(command):
+    words = nltk.word_tokenize(command)
+    taggedTokens = nltk.pos_tag(words)
```

Comment:
@526avijitgupta No, flake8 doesn't suggest camelCase.

@wimpykid26 Can you send a screenshot showing flake8 suggesting camelCase